content/4_Gold/Knapsack_DP.mdx

```diff
@@ -94,19 +96,19 @@ $\texttt{dp}[x]$ if $x < 0$.
 
 using namespace std;
 
-typedef long long ll;
+long long dp[1000001];
 
 int main() {
 	int n;
 	cin >> n;
-	ll dp[n+1]{}; dp[0] = 1;
-	for (int i = 1; i<= n; i++) {
-		for (int j = 1; j <= 6; j++) {
-			if (i-j >= 0) {
-				dp[i] += dp[i-j];
-				dp[i] %= 1000000007;
+	dp[0] = 1;
+	for(int i = 1; i <= n; i++) {
```

Comment:
Could you add spaces between stuff like `for`/`if` and the opening brackets please? I think that was a new addition to the style guide. Otherwise, it looks fine

Reply:
I don't think this was added to the style guide? Could you send a link to where it says this?

Reply:
Oh ok my bad